core/server/worker/src/main/java/alluxio/worker/netty/DataServerReadHandler.java

```diff
@@ -165,7 +166,9 @@
 
   @Override
   public void channelUnregistered(ChannelHandlerContext ctx) {
-    setError(ctx.channel(), new Error(null, false, Protocol.Status.Code.INTERNAL));
+    // TODO(andrew): talk to peis about whether this is the right exception type
```

Comment:
This message is not sent to the client (note the false in Error ctor). This is handled when the channel is closed. no way to send the information back.